src/write_outputs/ucommit/write_commit.jl

```diff
@@ -11,4 +11,18 @@ function write_commit(path::AbstractString, inputs::Dict, setup::Dict, EP::Model
     auxNew_Names = [Symbol("Resource"); Symbol("Zone"); [Symbol("t$t") for t in 1:T]]
     rename!(dfCommit, auxNew_Names)
     CSV.write(joinpath(path, "commit.csv"), dftranspose(dfCommit, false), header = false)
+    println("-------------------------------")
+    println("PATH = ", path)
+    println("-------------------------------")
+
+    if setup["OutputFullTimeSeries"] == 1 & setup["TimeDomainReduction"] == 1
+        DFMatrix = Matrix(dftranspose(dfCommit, true))
+        DFnames = DFMatrix[1, :]
+        FullTimeSeriesFolder = setup["OutputFullTimeSeriesFolder"]
+        output_path = joinpath(path, FullTimeSeriesFolder)
+        dfOut_full = full_time_series_reconstruction(
+            path, setup, dftranspose(dfCommit, false), DFnames)
+        CSV.write(joinpath(output_path, "commit.csv"), dfOut_full, header = false)
+        println("Writing Full Time Series for Commitment")
+    end
```

Comment:
This section could be wrapped in a function that is called in all the `write_*.jl` files, making it easier to maintain.